app/coffeescript/components/data/info_window.coffee

```diff
@@ -3,22 +3,20 @@
 define [
   'jquery'
   'flight/lib/component'
-  'map/components/mixins/map_utils'
 ], (
   $
   defineComponent
-  mapUtils
 ) ->
 
   infoWindowData = ->
 
     @defaultAttrs
       pinRoute: "/map/pin/"
-
+      pinRefinements: ''
```

Comment:
We can also remove the `pin` prefix. It avoids the Smurf Naming Convention ;)

![screenshot 2015-12-07 12 12 46](https://cloud.githubusercontent.com/assets/338738/11633922/dca4a876-9cdb-11e5-8135-2feaaeb59da4.png)